src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -30,11 +30,11 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresSe
                 const options = {
                     location: locationName,
                     sku: {
-                        name: "GP_Gen5_4",
-                        tier: "GeneralPurpose",
-                        capacity: 4,
+                        name: "B_Gen5_1",
+                        tier: "Basic",
+                        capacity: 1,
                         family: "Gen5",
-                        size: "102400"
+                        size: "5120"
```

Comment:
I think you mean `51200`

Also, let's make a constant for this since it's used twice

Reply:
The size variable takes values in MB and the lowest available size on portal is 5GB.

Reply:
Okay I get why we're picking cheaper options as the default, but do we really need the lowest of the low? 😝 5 GB seems really low to me, and it doesn't even save that much money. If we go that low, I propose we make sure 'storageAutoGrow' is set to 'Enabled'

Reply:
cc @fiveisprime @nturinski for your thoughts on storage size and/or [auto-grow](https://docs.microsoft.com/azure/postgresql/concepts-pricing-tiers#storage-auto-grow)

Reply:
Nathan and Matt agreed with Eric, let's use the portal's basic default which is 50 GB with auto-grow enabled.